### ISSUE TITLE
Add "edit /target/etc/crypttab for encrypted installations" to explanation page for expert install

### ISSUE THREAD

**Issue by harlekin777** [linuxmint/lmde-4-cinnamon-beta#49]

**Comment by harlekin777** [comment]
Live-installer 2020.03.07 expert mode with manual Luks setup finished install process successfully thanks to added "next"-button. But computer does not boot after installation. After Grub screen the following message appears:

BusyBox v1.30.1 (Debian 1:1.30.1-4) built-in shell (ash)
Enter 'help' for a list of built-in commands

(initramfs) exit
...........
...........
...........
ALERT! UUID=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx does not exist. Dropping to a shell!

Cryptsetup encryption password dialog does not appear as expected.


My custom /etc/fstab file:

proc    /proc   proc    defaults        0   0
UUID=C63D-924B  /boot/efi   vfat    defaults    0   2
UUID=af42dadf-d975-4bec-900e-bbaff367e5c3   /boot   ext4    defaults    0   2       
/dev/mapper/sda3_crypt   swap    swap    defaults    0   2
/dev/mapper/sda8_crypt   /   ext4    defaults    0   1

There was an error message after installing mdadm; about sda3_crypt and sda8_crypt not appearing in /etc/crypttab.

/etc/crypttab was empty. Editing it did not help, either.

My edited /etc/crypttab file (did not work):

sda3_crypt   UUID=1a8315a9-b6e2-41ba-ab9f-d20db5c5da78   none
sda8_crypt   UUID=473b6e25-59bf-430e-98a7-575fca2c2766   none

Standard installation on same machine without problems.

**Comment by monsta** [comment]
Just in case - did you mount the unencrypted `/boot` partition to `/target/boot` before proceeding?

**Comment by monsta** [comment]
Also maybe add `luks` option (after `none`) to the crypttab entries. I recall that the automated installation does it, and so does Calamares in LMDE 3.

**Comment by harlekin777** [comment]
Thank you very much! Of course I had mounted unencrypted /boot to /target/boot and /boot/efi to /target/boot/efi. Your hint regarding luks option in /etc/crypttab did the trick. I had not included it before, because some documentation said, it is not a mandatory field. But in this case it is required. It works now.

In any case it is absolutely necessary to edit the /target/etc/crypttab file before (!) chrooting and installing mdadm (i.e. before updating initramfs). Therefore I strongly recommend to mention crypttab in the explanation page, when the install pauses, in this way:
- edit /target/etc/fstab
- edit /target/etc/crypttab
- chroot to /target and install mdadm etc.

It would be helpful for others, if you could add it. Thank you again!

**Comment by monsta** [comment]
I agree, it can be mentioned, with additional note that it's only needed for encrypted installations.

**Comment by harlekin777** [comment]
Sure. Thanks!

**Comment by harlekin777** [comment]
@monsta would it be possible to add this helpful remark?

**Comment by monsta** [comment]
Yes, but for LMDE 5. You can add an issue for live-installer so we won't forget about it...
